DEVELOPMENT.md

```diff
@@ -269,3 +269,125 @@ If you need to add a new CRD type, you will need to add:
    - [200-clusterrole.yaml](./config/200-clusterrole.yaml)
    - [clusterrole-aggregate-edit.yaml](./config/clusterrole-aggregate-edit.yaml)
    - [clusterrole-aggregate-view.yaml](./config/clusterrole-aggregate-view.yaml)
+
+## Adding feature gated API fields
+
+We've introduced a feature-flag called `enable-api-fields` to the
```

Comment:
How does this flag relate to API versions? e.g. what happens if enable-api-fields = alpha for the beta API or enable-api-fields = stable for the alpha API?

(I suspect this might just be a naming issue - happy to bikeshed alternatives!)

Reply:
(will update the docs)